TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -859,38 +981,38 @@ std::string TTSSpeaker::constructURL(TTSConfiguration &config, SpeechData &d) {
     std::string tts_request;
     TTSURLConstructer url;
     tts_request = url.constructURL(m_defaultConfig,d.text,false);
```

Comment:
here the variable `url` is little misleading, 'urlConstructor' would be more suitable also we shall stick to the uniform naming conventions